pkg/controller/authentication/job.go

```diff
@@ -211,7 +211,7 @@ func buildContainer(jobName string, image string, resources *corev1.ResourceRequ
 		{
 			Name:            jobName,
 			Image:           image,
-			ImagePullPolicy: corev1.PullAlways,
+			ImagePullPolicy: corev1.PullIfNotPresent,
```

Comment:
is this the same for all images?

Reply:
I have made it for the im-initcontainer image, as this is just plain ubi-minimal image, we don't really need Always pullPolicy